src/js/worklets/shell.js

```diff
@@ -18,11 +18,11 @@ export function stackOpacity (stackId, selectedStackId) {
   );
 }
 
-export function stackPointer (stackId, selectedStackId) {
+export function stackZIndex (stackId, selectedStackId) {
   return useDerivedValue(
     function () {
       'worklet'
-      return selectedStackId.value == stackId ? "auto" : "none";
+      return selectedStackId.value == stackId ? 10 : 9;
```

Comment:
this seems like something that might benefit from having a comment explaining. Or at least put `10` and `9` in some variables that describe what is going on here. 
Curious of why you selected 10 and 9? 🤔